examples/fluids/navierstokes.c

```diff
@@ -1153,6 +1173,47 @@ int main(int argc, char **argv) {
   CHKERRQ(ierr);
   ierr = PetscOptionsEnd(); CHKERRQ(ierr);
 
+  // Setup BCs for Rotation or Uniform wind types in Advection (3d)
+  if (problemChoice == NS_ADVECTION) {
+    switch (wind_type) {
+    case ADVECTION_WIND_ROTATION:
+      // No in/out-flow
+      bc.ninflow = bc.noutflow = 0;
+      break;
+    case ADVECTION_WIND_UNIFORM:
+      // Face 6 is inflow and Face 5 is outflow
+      bc.ninflow = bc.noutflow = 1;
+      bc.inflow[0] = 6; bc.outflow[0] = 5;
+      // Faces 3 and 4 are slip
+      bc.nslip[0] = bc.nslip[2] = 0; bc.nslip[1] = 2;
+      bc.slips[1][0] = 3; bc.slips[1][1] = 4;
+      // Faces 1 and 2 are wall
+      bc.nwall = 2;
+      bc.walls[0] = 1; bc.walls[1] = 2;
+      break;
+    }
+  }
```

Comment:
@valeriabarra , default BCs are here.

Reply:
So, you don't need to specify BCs.

Reply:
If you make the flow diagonal, there will be two inflow faces and two outflow faces. Have you been doing that or just running with default parameters?

Reply:
Ah, so that makes it strange. No, I haven't. I thought I was only changing the velocity vector. I will fix it. Thanks.